Digipost/APIClient.swift

```
@@ -433,9 +381,15 @@ class APIClient : NSObject, NSURLSessionTaskDelegate, NSURLSessionDelegate, NSUR
                         self.deleteRefreshTokensAndLogoutUser()
                     }
             })
-        }else if (oAuthToken == nil) {
+        } else if (oAuthToken == nil) {
             assert(false," something wrong with oauth token")
```

If this is something that will never happen we should fail harder.